### DISCUSSION TITLE
Feature request - Define filters on relation fields

### DISCUSSION THREAD

**Discussion by thinkbig1979** [twentyhq/twenty#7144] [Ideas]
Let's say you have a 'Building' object, and you have design firms in your Companies list. Now you want to link the designer of the building to the building itself. You can create a relation field called "Designer" in the Building object, and link it to Companies, no problem.

But then when you go to link a design firm to a building, you have to select from all the companies in you database. 
This could be very elegantly solved by allowing the admin to assign one or more filters to that relation field. 

E.g. You could add a select field called 'Category' to your companies, and have a category called "Design Firm" . If you could pre-filter on that category in the relation field, you would only see relevant entries when going to add the designer. 

Just a really nice quality of life improvement.

**Comment by Bonapara** [comment]
Hi @thinkbig1979, I love it, this is indeed a great quality-of-life feature. Thanks for raising this issue! We won't implement it in the short term, though, as it's an advanced settings feature and we're still focused on the basics. Converting it into a discussion.

**Comment by BOHEUS** [comment]
It's planned feature for Q2 26 https://github.com/twentyhq/core-team-issues/issues/213
